tests/lib/prepare-restore.sh

```diff
@@ -585,7 +585,9 @@ prepare_project() {
     # go mod runs as root and will leave strange permissions
     chown test:test -R "$SPREAD_PATH"
 
-    if [ "$BUILD_SNAPD_FROM_CURRENT" = true ]; then
+    if tests.info is-snapd-pkg-repo; then
+        ( cd "${GOHOME}" && tests.pkgs download snapd snap-confine ubuntu-core-launcher)
```

Comment:
I think we can drop ubuntu-core-launcher from this list unless some old system needs matching versions of all the packages. Does it work without it?

Reply:
I think is it not needed, I'll try removing it.